tests/test_recipes.py

```diff
@@ -158,6 +159,22 @@ def test_key(self):
         self.assertTrue(mi.all_equal('4٤໔４৪', key=int))
         self.assertFalse(mi.all_equal('Abc', key=str.casefold))
 
+    @patch('more_itertools.recipes.groupby', autospec=True)
```

Comment:
It feels like the mock object's `call_count` should be able to be used here, but I couldn't get it to actually work.